File: Java/mediation-bidder/src/main/java/vdo/ai/sample/java/mediation/bidder/AdManagerHomeActivity.java
package vdo.ai.sample.java.mediation.bidder;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;

import vdo.ai.sample.java.mediation.bidder.AdManager.BannerActivity;
import vdo.ai.sample.java.mediation.bidder.AdManager.InterstitialActivity;
import vdo.ai.sample.java.mediation.bidder.AdManager.RewardedInterstitialActivity;
import vdo.ai.sample.java.mediation.bidder.AdManager.RewardedVideoActivity;
import vdo.ai.sample.java.mediation.bidder.databinding.ActivityHomeBinding;

public class AdManagerHomeActivity extends AppCompatActivity {

    ActivityHomeBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityHomeBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.main, menu);
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        int itemId = item.getItemId();
        if (itemId == R.id.banner_ad) {
            startActivity(new Intent(this, BannerActivity.class));
        } else if (itemId == R.id.interstitialAd) {
            startActivity(new Intent(this, InterstitialActivity.class));
        } else if (itemId == R.id.rewarded_videoAd) {
            startActivity(new Intent(this, RewardedVideoActivity.class));
        } else if (itemId == R.id.rewarded_interstitialAd) {
            startActivity(new Intent(this, RewardedInterstitialActivity.class));
        }
        return super.onOptionsItemSelected(item);
    }
}

File: Kotlin/example-mediation-bidder/src/main/java/vdo/ai/android/example/mediation/bidder/AdManagerHomeActivity.kt
package vdo.ai.android.example.mediation.bidder

import android.content.Intent
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import vdo.ai.android.example.mediation.bidder.AdManager.BannerActivity
import vdo.ai.android.example.mediation.bidder.AdManager.InterstitialActivity
import vdo.ai.android.example.mediation.bidder.AdManager.RewardedVideoActivity
import vdo.ai.android.example.mediation.bidder.AdManager.RewardedInterstitialActivity

import vdo.ai.android.example.mediation.bidder.databinding.ActivityHomeBinding

class AdManagerHomeActivity : AppCompatActivity() {
    var binding: ActivityHomeBinding? = null
    protected override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityHomeBinding.inflate(layoutInflater)
        setContentView(binding?.root)
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        getMenuInflater().inflate(R.menu.main, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        val itemId = item.itemId
        if (itemId == R.id.banner_ad) {
            startActivity(Intent(this, BannerActivity::class.java))
        } else if (itemId == R.id.interstitialAd) {
            startActivity(Intent(this, InterstitialActivity::class.java))
        } else if (itemId == R.id.rewarded_videoAd) {
            startActivity(Intent(this, RewardedVideoActivity::class.java))
        } else if (itemId == R.id.rewarded_interstitialAd) {
            startActivity(Intent(this, RewardedInterstitialActivity::class.java))
        }
        return super.onOptionsItemSelected(item)
    }
}